src/topology/compact_open.lean

```diff
@@ -103,6 +103,24 @@ lemma continuous_comp_left : continuous (λ g, g.comp f : C(β, γ) → C(α, γ
 continuous_generated_from $ assume m ⟨s, hs, u, hu, hm⟩,
   by { rw [hm, image_gen f hs hu], exact continuous_map.is_open_gen (hs.image f.2) hu }
 
+/-- Composition is a continuous map from `C(α, β) × C(β, γ)` to `C(α, γ)`, provided that `β` is
+  locally compact. This is Prop. 9 of Chap. X, §3, №. 4 of Bourbaki's *Topologie Générale*. -/
+lemma continuous_prod (α β γ : Type*) [topological_space α] [topological_space β]
```

Comment:
`continuous_comp`? Also, could you please add corollaries similar to `continuous.mul`/`filter.tendsto.mul`? I'm not sure what should be the operation name for dot notation here: we can't use just `comp` because we may want lemmas like this for compositions of other bundled maps (e.g., `continuous_linear_map`s) too.

Reply:
Changed the name, but for the corollaries I am not sure to understand your hint: something like
```
lemma foo (α β γ : Type*) [topological_space α] [topological_space β]
  [locally_compact_space β] [topological_space γ] [has_mul γ] [has_continuous_mul γ]:
  continuous (λ x y : C(α, β) × C(β, γ), (x.2.comp x.1) * (y.2.comp y.1))
```
(with the problem that `has_continuous_mul γ` is not enough to get a `continuous_mul` on `C(α, β)`)?

Reply:
I think this is the sort of lemma that @urkud has in mind:
```lean
lemma continuous.comp' {X α β γ : Type*} [topological_space X]
  [topological_space α] [topological_space β]
  [locally_compact_space β] [topological_space γ] {f : X → C(α, β)} {g : X → C(β, γ)}
   (hf : continuous f) (hg : continuous g) :
  continuous (λ x, (g x).comp (f x)) :=
(continuous_prod α β γ).comp (hf.prod_mk hg : continuous (λ x, (f x, g x)))
```
(see also https://leanprover-community.github.io/mathlib_docs/notes.html#continuity%20lemma%20statement)

As regards naming, how about `continuous_comp'` instead of `continuous_prod`?

Reply:
Thanks, done!